### PR TITLE
CI check script updates

### PR DESCRIPTION
**Pull request by dhaiducek**
- Drop `2.4`
- Drop `grc-ui`/`grc-ui-api`
- Add policy generator and policy collection

### CODE REVIEW

File: build/periodic.sh [hunk 1]
```diff
@@ -22,11 +22,11 @@ cloneRepos() {
...
 		# Manually append deprecated repos
 		REPOS="${REPOS}
-			stolostron/grc-ui
-			stolostron/grc-ui-api
 			stolostron/governance-policy-spec-sync
 			stolostron/governance-policy-status-sync
-			stolostron/governance-policy-template-sync"
+			stolostron/governance-policy-template-sync
+			stolostron/policy-collection
+			stolostron/policy-generator-plugin"
```
Comment (mprahl) [inline build/periodic.sh:29]: Should we also add `governance-policy-framework-addon`?

Comment (dhaiducek) [reply build/periodic.sh:29]: No. It's already in `repo.txt`: https://github.com/stolostron/policy-grc-squad/blob/35f8a8646b751ce8a8039b938a4e3dbee3aad051/main-branch-sync/repo.txt#L6

Comment (dhaiducek) [reply build/periodic.sh:29]: These are the repos that either don't have an image build or have dropped from our current version.

**Pull request merged**